src/test-utils/providers.tsx

```diff
@@ -0,0 +1,16 @@
+import { ReactNode } from 'react'
+import { AuthStoreProvider } from '@/lib/providers/auth-store-provider'
+import { OrganizationStateStoreProvider } from '@/lib/providers/organization-state-store-provider'
+import { CoachingRelationshipStateStoreProvider } from '@/lib/providers/coaching-relationship-state-store-provider'
+
+export function TestProviders({ children }: { children: ReactNode }) {
```

Comment:
It wasn't obvious until I saw this file what `TestProviders` was accomplishing. Now it makes total sense. 
I wonder if it would make sense to just provide an application level aggregation of these that can be used in the app _and_ in the tests? Like a `Providers`?

Reply:
A really sharp observation and definitely worth doing.